README.md

```diff
@@ -63,10 +63,10 @@ Do you like Apache Maven? Then [donate back to the ASF](https://www.apache.org/f
 
 Quick Build
 -------
-If you want to bootstrap Maven, you'll need:
+If you want to bootstrap Maven from source, you'll need:
 - Java 1.8+
 - Maven 3.0.5 or later
-- Run Maven, specifying a location into which the completed Maven distro should be installed:
+- Run Maven already installed, specifying a location into which the completed Maven distro should be installed:
```

Comment:
Since an already installed Maven is required in the line above, this change adds not new information.